src/test/java/homework14/SeleniumWebDriver.java

```diff
@@ -0,0 +1,142 @@
+package homework14;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+
+import java.time.Duration;
+
+import static org.junit.Assert.assertEquals;
+
+public class SeleniumWebDriver {
+    public static final By ELEMENTS_CARD = By.xpath("/html/body/div[2]/div/div/div[2]/div/div[1]/div/div[3]/h5");
+    public static final By RADIO_BUTTON_MENU_ITEM = By.id("item-2");
+    public static final By RADIO_BUTTON_COMMON_MESSAGE_RESULT = By.xpath("/html/body/div[2]/div/div/div/div[2]/div[2]/p");
```

Comment:
Fix the locator. Do not use xpath with absolute path `/html/body/div[2]/div/div/div/div[2]/div[2]/p`